cmd/secrets-provider/main.go

```diff
@@ -130,11 +130,12 @@ func secretRetriever(ctx context.Context,
 		log.Error(messages.CSPFK008E)
 		return nil, err
 	}
-	if err = validateContainerMode(authnConfig.GetContainerMode()); err != nil {
+	// Do we need to check this? Container mode is checked in ValidateAnnotations
```

Comment:
Does `ValidateAnnotations` catch if the container mode annotation is missing? Or is that OK since it'll just use the default?

Reply:
I have removed validateContainerMode as `ValidateAnnotations` checks this.